e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/dashboard-deals-filter-by-multiple-fields-multiple-values.spec.js

```diff
@@ -44,34 +61,27 @@ context('Dashboard Deals filters - filter by multiple fields with multiple value
 
   before(() => {
     cy.login(BANK1_MAKER1);
-    dashboardDeals.visit();
-    cy.url().should('eq', relative('/dashboard/deals/0'));
   });
 
-  it('submits the filters and redirects to the dashboard', () => {
+  beforeEach(() => {
+    cy.saveSession();
+    dashboardDeals.visit();
+
     // toggle to show filters (hidden by default)
     filters.showHideButton().click();
+  });
 
-    // apply filter 1
-    dashboardDeals.filters.panel.form.status.draft.checkbox().click();
-
-    // apply filter 2
-    dashboardDeals.filters.panel.form.status.readyForChecker.checkbox().click();
-
-    // apply filter 3
-    dashboardDeals.filters.panel.form.submissionType.MIA.checkbox().click();
-
-    // apply filter 4
-    dashboardDeals.filters.panel.form.submissionType.AIN.checkbox().click();
-
-    // submit filters
-    filters.panel.form.applyFiltersButton().click();
-
+  it('Should visit correct page', () => {
```

Comment:
Not a blocker and very minor - seems the naming convention is lowercase

Reply:
Few instances of this

Reply:
Renamed.